README.md

```diff
@@ -1,36 +1,88 @@
-## Overview
+## DotNetLightning: The utility for working with the Bitcoin Lightning Network (LN) in C#, F#
 
-This repository contains one solution which contains four main projects.
+the main entry point is `DotNetLightning.Core`.
 
-1. `DotNetLightning.Core` to perform LN specific logic and low level primitives.
-2. `DotNetLightning.Infrastructure` to support data persistency, altcoins, channel/peer management etc.
-3. `DotNetLightning.Server` to run as a server.
-4. `DotNetLightning.Client` to interact with the server.
+## Installation
 
-## How to configure
+The package is compiled and published with two variants
 
-Configuration will follow an usual semantics for ASP.NET Core web app.
-That is, it will read in following order.
+* [`DotNetLightning`](https://www.nuget.org/packages/DotNetLightning/)
+  * This does not use native bindings for cryptographic operation.
+  * This is the one you want to use if you run your code everywhere, but possibly slower then below.
+* [`DotNetLightning.Core`](https://www.nuget.org/packages/DotNetLightning.Core/)
+  * This uses pre-compiled `libsodium` for cryptographic operation.
+  * It only supports `windows`, `mac` and `linux` environment.
+  * This is what you want if you need performance and above are the only environments you are planning to support.
```

Comment:
and above are the only environments you are planning to support. -> and the environments above are the only ones you are planning to support.